app/app.go

```diff
@@ -714,3 +761,12 @@ func (a *app) SetDeviceStateIntegration(
 func (a *app) GetEvents(ctx context.Context, filter model.EventsFilter) ([]model.Event, error) {
 	return a.store.GetEvents(ctx, filter)
 }
+
+func runAndLogError(ctx context.Context, f func() error) {
+	err := f()
+	if err != nil {
+		log.FromContext(ctx).
+			WithField("panic", err.Error()).
```

Comment:
The `panic` log fields are used for logging recovered panics in the access log middleware, could you please change it to `error`?
```suggestion
			WithField("error", err.Error()).
```